packages/stream/__tests__/streams.spec.ts

```diff
@@ -0,0 +1,165 @@
+import StreamClient from "../StreamClient";
+import {
+  Cluster,
+  CreateMultiParams,
+  CreateMultipleStreamsValues,
+  LocalCluster,
+  TransferCancelOptions,
+} from "../types";
+import * as web3 from "@solana/web3.js";
+import { getBN } from "../utils";
+import { requestAirdrop } from "../airdrop-request";
+import { Wallet } from "@project-serum/anchor";
+import { PublicKey } from "@solana/web3.js";
+import { AIRDROP_TEST_TOKEN } from "../constants";
+import { BN } from "bn.js";
+let streamInstance: StreamClient | null = null;
+let sender: Wallet;
+let recipient: Wallet;
+let recipients: Wallet[] = [];
+let connection: web3.Connection | null = null;
+import { add, addMinutes, format, getUnixTime } from "date-fns";
+const CLUSTER_LOCAL = "local";
+
+const clusterUrls: { [s: string]: () => string } = {
+  [CLUSTER_LOCAL]: () => "http://localhost:8899", // http://127.0.0.1:8899",
+  [Cluster.Devnet]: () => "https://api.devnet.solana.com",
+  [Cluster.Mainnet]: () =>
+    "https://streamflow.rpcpool.com/8527ad85d20c2f0e6c37b026cab0",
```

Comment:
this url is stale, not working anymore, 8527ad85d20c2f0e6c37b026cab0 was a private token that shouldn't have been leaked (it was, before, i'm aware)
, just an FYI for future, so we don't accidentaly share these

Reply:
Ok, will change it to usual mainnet